internal/flags/apiflags/flags.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiflags
+
+import (
+	"fmt"
+	"strings"
+
+	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+	"github.com/spf13/pflag"
+)
+
+// APIFlags ...
```

Comment:
Nit: Remove the comment (okay, since we're in an internal package) or improve based on [godoc conventions](https://blog.golang.org/godoc).